keyboards/crkbd/keymaps/via_mechboards/readme.md

```diff
@@ -0,0 +1,12 @@
+# Default Via keymap for the Corne R2G by Mechboards UK
+
+![r2g](https://cdn.shopify.com/s/files/1/0582/0242/3501/products/HelidoxCorneR2GPCB_1800x1800.png)
```

Comment:
Could this be rehosted on Imgur.

Reply:
Opted to delete rather than rehost, not sure the image adds much.